ci-operator/config/codeready-toolchain/toolchain-e2e/codeready-toolchain-toolchain-e2e-master.yaml

```diff
@@ -53,6 +53,6 @@ tests:
   container:
     from: src
 - as: e2e
-  commands: make test-e2e
+  commands: IMAGE_HOST_OPERATOR=${IMAGE_HOST_OPERATOR} IMAGE_MEMBER_OPERATOR=${IMAGE_MEMBER_OPERATOR} IMAGE_REGISTRATION_SERVICE=${IMAGE_REGISTRATION_SERVICE} make test-e2e
```

Comment:
the value here is repeated.

Reply:
I don't understand - this is a different job that the other ones - I need to build all three images to make this job running.